sdks/java/harness/src/main/java/org/apache/beam/fn/harness/data/PCollectionConsumerRegistry.java

```diff
@@ -18,6 +18,7 @@
 package org.apache.beam.fn.harness.data;
 
 import com.google.auto.value.AutoValue;
+import com.google.common.collect.ImmutableList;
```

Comment:
Use our vendored guava. Caught by dependency analysis:

```
06:58:02 Execution failed for task ':sdks:java:harness:analyzeClassesDependencies'.
06:58:02 > Dependency analysis found issues.
06:58:02   usedUndeclaredArtifacts: 
06:58:02    - com.google.guava:guava:30.1-jre@jar
06:58:02 
```

And this is also the checkstyle failure.

Reply:
gah, intellij always pulls the wrong ones in for me. Fixed